sway-lib-std/src/context.sw

```diff
@@ -1,11 +1,11 @@
 //! Functionality for accessing context-specific information about the current contract or message.
 library context;
-dep context/registers;
-dep context/call_frames;
+dep registers;
+dep call_frames;
```

Comment:
Are those still needed?

Reply:
I thought so, but they are not. Will remove and update a few more paths.